server/server.ts

```diff
@@ -214,6 +214,21 @@ const generateToken = async (
   const jwtToken = await sign(payload, jwtSecret)
   return jwtToken
 }
+
```

Comment:
hmm a separate route to handle if .env is setup correctly or not.
any alternative like in the get user call itself sending metadata about the setup if user is admin role and above?